### DISCUSSION TITLE
Using MotionPlus with MotionEye

### DISCUSSION THREAD

**Discussion by CHARL13is** [Motion-Project/motionplus#132] [Q&A]
Hi all

I've been having issues with my setup [https://github.com/motioneye-project/motioneye/issues/2948](url) and I'm hopeful that MotionPlus is an answer.  Has anyone successfully used MotionPlus with MotionEye?  If so is there a guide somewhere on how to retrofit it?

Thanks

**Comment by Mr-Dave** [comment]
I wouldn't be optimistic that Motionplus could be a backend to Motioneye.  Motionplus changes some of the parameters as well as the stream port(s).   I'd think that if you want to get that Pi cam working you'd either:
1.  Need to find a way to get that tuning json into "libcamerify" (I'm not sure if that is possible or if so how to do it), modify Motioneye to then use the modified command line (e.g. Change the motion binary specification inside the Motioneye configuration).
2. Or, use Motionplus directly and convert the tuning parameters into libcam_params.  Next would be to add whichever scripts you wanted to the Motionplus configuration files that are part of Motioneye.

One question that I'd have for you on this is what is the desired functionality provided by Motioneye?

**Reply by CHARL13is** [reply]
Apologise for the delayed response @Mr-Dave, I was on leave last week.  I've managed to pull something together by outputting libcamerify to RTSP and then consuming that through Motioneye.  As you might imagine it's not particularly performant but it will do the job for now.

My goal is to present a camera feed to an end user via a webpage, as well as being able to capture video when motion is detected.  When I did the same project last year Motioneye fit the bill nicely, however I've been caught out this year by a change in hardware and OS!
